docs/src/css/custom.css

```
@@ -0,0 +1,30 @@
+/**
+ * Any CSS included here will be global. The classic template
+ * bundles Infima by default. Infima is a CSS framework designed to
+ * work well for content-centric websites.
+ */
+
+/* You can override the default Infima variables here. */
+:root {
+  --ifm-color-primary: #2e8555;
```

To fit with our theme better, similar shades to our main website.

```suggestion
  --ifm-color-primary: #4e33ff;
```